js/analyzer.js

```diff
@@ -173,7 +173,8 @@ YoastSEO.Analyzer.prototype.addAnalysis = function( analysis ) {
  * @returns {{test: string, result: (Function|YoastSEO.PreProcessor.wordcount|Number)}[]}
  */
 YoastSEO.Analyzer.prototype.wordCount = function() {
-	return [ { test: "wordCount", result: this.preProcessor.__store.wordcountNoTags } ];
+	var countWords = require( "./stringProcessing/countWords.js" );
+	return [ { test: "wordCount", result: countWords( this.preProcessor.__store.cleanTextNoTags ) } ];
```

Comment:
I thought the preprocessor could be removed after these changes, we should at least be able to remove it completely from the analyser.